mods/ra/rules/structures.yaml

```diff
@@ -338,6 +340,21 @@ PDOX:
 		Duration: 20
 		KillCargo: yes
 		DisplayRadarPing: True
+	ChronoshiftPower@advancedchronoshift:
+		OrderName: AdvancedChronoshift
+		Prerequisites: structures.germany
```

Comment:
Providing the prerequisite through the player means that non-german factions will not be able to capture an advanced chronosphere, and that chronospheres captured by a german player will automatically upgrade to the advanced version. The same goes for the soviet factions' super powers. This behaviour is inconsistent with the french gap generator, so I think it should probably get the same treatment. A general question, though: do we not want this tech to be captureable?